
<launch>  
  <node name="ndt_mcl" pkg="ndt_localization" type="ndt_mcl_localization_filter" output="screen">
    <!-- jff file with map -->
    <param name="map_file" value="$(find ndt_fuser)/maps/basement3d_map.jff" />
    <param name="sensor_pose_x" value="0.3" />
    <param name="sensor_pose_z" value="1.3" />
    <param name="sensor_pose_t" value="-1.62" />
    
    <param name="pose_init_x" value="10.73" />
    <param name="pose_init_y" value="2.185" />
    <param name="pose_init_t" value="0.05" />

    <!-- resolution of jff map -->
    <param name="resolution" value="0.4" />
    <!-- show aprticles for debug purposes -->
    <param name="show_particles" value="true" />
    <!-- particle count for ndt-mcl -->
    <param name="particle_count" value="300" />
    <param name="show_pose" value="true"/>
    <param name="fraction" value="0.6"/>
    <param name="force_SIR" value="true" />
    <param name="input_topic_name" value="velodyne_points" />
    <param name="odom_topic_name" value="/vmc_navserver/odom" />
    <param name="PointCloud" value="true" />
    <param name="odom_tf" value="world" />
    <param name="root_tf" value="world" />
    <param name="base_tf" value="odom_base_link" />
  </node>

  
</launch>
